README.md

```diff
@@ -171,7 +171,7 @@ In the following example, we can see that the recommended number of replicas is
 * **Scaling Delay**
 <a name="delay"></a>
 
-In order to avoid scaling from bursts you can use the following features: `downscaleDelayBelowWatermarkSeconds` and/or `upscaleDelayAboveWatermarkSeconds`. These options are specified as integers. The metric has to remain above or under its respective watermark for the configured duration.
+In order to avoid scaling from bursts you can use the following features: `downscaleDelayBelowWatermarkSeconds` and/or `upscaleDelayAboveWatermarkSeconds`. These options are specified as integers. The metric(s) have to remain above or under its/their respective watermark for the configured duration.
```

Comment:
This was better phrased before, in terms of translatability :)
```suggestion
In order to avoid scaling from bursts you can use the following features: `downscaleDelayBelowWatermarkSeconds` and/or `upscaleDelayAboveWatermarkSeconds`. These options are specified as integers. Each metric has to remain above or under its respective watermark for the configured duration.
```

Reply:
With `Each` my understanding is that we use `AND` and need all the metrics to stay above/below for the duration whereas we only need the union of the metrics to be above/below.